proto/stat_var.proto

```diff
@@ -232,4 +232,22 @@ message GetStatVarSummaryRequest {
 }
 message GetStatVarSummaryResponse {
   map<string, StatVarSummary> stat_var_summary = 1;
+}
+
+message GetStatVarMatchRequest {
+  // The property value pair. Ex
+  // {gender: Male}, {statType: measuredValue}
+  // To improve this, can explicitly specify "statType", "measuredProperty",
+  // and constraint properties. Then the match could have must-match terms and
+  // optional match terms that have weights. That would greatly reduce the
+  // number of returned match stat vars.
+  map<string, string> property_value = 1;
+}
+
+message GetStatVarMatchResponse {
+  message MatchInfo {
+    string stat_var = 1;
+    int32 match_count = 2;
```

Comment:
How this is computed? add a comment maybe?

Reply:
I will submit this for now. Once we have a better idea of partial match, I will send a follow up PR.